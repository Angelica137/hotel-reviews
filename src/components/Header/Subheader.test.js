describe("the <Subheader /> component", () => {
  it("should render", () => {
    expect(1 + 2).toBe(3);
  });
});
