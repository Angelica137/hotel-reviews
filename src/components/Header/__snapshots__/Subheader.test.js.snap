// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the <Subheader /> component should render 1`] = `ShallowWrapper {}`;

exports[`the <Subheader /> component should render with a form button 1`] = `ShallowWrapper {}`;
